.github/workflows/docs.yml

```diff
@@ -24,14 +28,16 @@ jobs:
 
       - name: Build
         run: |
-          if [ "${{ steps.setup.outputs.IS_RELEASE }}" == "true" ]; then
+          if [ "${STEPS_SETUP_OUTPUTS_IS_RELEASE}" == "true" ]; then
             O="-t release"
           fi
           make html O="$O"
+        env:
+          STEPS_SETUP_OUTPUTS_IS_RELEASE: ${{ steps.setup.outputs.IS_RELEASE }}
```

Comment:
I think we can change the `setup` step to write directly to `$GITHUB_ENV` to set the environment variable in the step instead of later here.